src/robusta/core/playbooks/internal/crds.py

```
@@ -0,0 +1,360 @@
+            crd_list.append(crd_info)
+
+        crd_list.sort(key=lambda x: x.get("kind", ""))
+
+        json_block = JsonBlock(
+            json_str=json.dumps(crd_list, indent=2)
+        )
+
+        finding.add_enrichment([json_block])
+        event.add_finding(finding)
+
+    except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)